src/main/java/com/dd/plist/BinaryPropertyListWriter.java

```diff
@@ -111,7 +111,7 @@ public static void write(File file, NSObject root) throws IOException {
     public static void write(OutputStream out, NSObject root) throws IOException {
         int minVersion = getMinimumRequiredVersion(root);
         if (minVersion > VERSION_00) {
-            String versionString = ((minVersion == VERSION_10) ? "v1.0" : ((minVersion == VERSION_15) ? "v1.5" : ((minVersion == VERSION_20) ? "v2.0" : "v0.0")));
+            String versionString = minVersion == VERSION_10 ? "v1.0" : ((minVersion == VERSION_15) ? "v1.5" : ((minVersion == VERSION_20) ? "v2.0" : "v0.0"));
```

Comment:
The parenthesis around `minVersion` == VERSION_15` and `minVersion == VERSION_20` are also useless